src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -132,7 +132,6 @@
   --pf-c-toolbar--child--MarginLeft: var(--pf-c-toolbar__action-list--MarginLeft);
 }
 
-.pf-c-toolbar__pagination,
```

Comment:
this will make it breaking now @mcoker

Reply:
will it though? the current toolbar top pagination is `.pf-c-pagination.pf-c-toolbar__pagination`. Seems like you could remove that selector and the `.pf-c-pagination` selector will still position the pagination component.

Reply:
Oh yeah the functionality is there so it should be good! Didn't see those classes, removed them.

Reply:
Ah... removing the classes from the elements will be a breaking change. If someone was using them to style the pagination, they're no longer available. I was just referring to removing `.pf-c-toolbar__pagination` from the CSS that applies `margin-left: auto` and only using `.pf-c-pagination` there. That shouldn't introduce a breaking change. We could create a follow-up issue to remove `.pf-c-toolbar_pagination` from those elements later though.

Reply:
Ohh ok so ill go back to what I had and then create a follow up issue